javascript-data-model/order-history.js

```diff
@@ -6,14 +6,16 @@ var orderHistory = [
     shipTo: 'JS Masher',
     orderNumber: 11439416898772232,
     items:
-    {
-      deliveryDate: 20200808,
-      deliveryMethod: null,
-      name: 'JavaScript for impatient programmers',
-      author: 'Ruschmayer, Dr. Axel',
-      returnCloseDate: 20200907,
-      price: 31.55
-    }
+    [
+      {
+        deliveryDate: 20200808,
+        deliveryMethod: null,
```

Comment:
These properties should be on the main order object, since they relate to the order, not to the individual items